api-s3-datatypes.go

```diff
@@ -128,7 +128,7 @@ type initiator struct {
 // copyObjectResult container for copy object response.
 type copyObjectResult struct {
 	ETag         string
-	LastModified string // time string format "2006-01-02T15:04:05.000Z"
+	LastModified time.Time // time string format "2006-01-02T15:04:05.000Z"
```

Comment:
How would `xmlDecoder()` know the format (time layout) with which to parse the time? I am not sure the decoding will be correct as is.

Reply:
that is the feature of xml Marshal @donatello also we need to have typed value.

Reply:
For example look all the List() APIs.

Reply:
@harshavardhana - xml decoder is using a particular format within the ISO8601 standard by default to parse timestamps. Looks like by dumb luck AWS is using the same format!